ui/pages/swaps/prepare-swap-page/not-enough-balance.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  SEVERITIES,
+  TextVariant,
+  Size,
+} from '../../../helpers/constants/design-system';
+import {
+  BannerAlert,
+  ButtonLink,
+  Text,
+} from '../../../components/component-library';
+
+interface Props {
+  title: string;
+  actionableBalanceErrorMessage: string;
+  needsMoreGas: boolean;
+  openBuyCryptoInPdapp: () => void;
+  needsMoreGasText: string;
+}
+
+export default function NotEnoughBalance({
+  title,
+  actionableBalanceErrorMessage,
+  needsMoreGas,
+  openBuyCryptoInPdapp,
+  needsMoreGasText,
+}: Props) {
+  return (
+    <BannerAlert severity={SEVERITIES.INFO} title={title} className="">
```

Comment:
Can we add a `className` here or remove this empty string?

Reply:
Cannot remove the empty string as it's a required prop in the `PropType` and TS is enforcing it. Didn't want to modify the underlying component and make this PR bloated and touching too many thing. I could if you think it's a good idea though.

Reply:
Or I can add a `className` with just empty CSS.

Reply:
Resolved in 9778cd2